src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+  
+  span.iui-left {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  span.iui-right {
+    margin-left: $iui-s;
+  }
+}
+
+@mixin iui-slider-container {
+  display: flex;
```

Comment:
Is flex actually needed? I removed it and visually nothing changed...
If not needed, we could get rid of flex-direction and justify-content too (they do nothing also...)

Reply:
You are correct, once I changed to use position absolute I should have changed this.